### ISSUE TITLE
Add a CONTRIBUTING.md file that explains versioning and release process

### ISSUE THREAD

**Issue by zarino** [mysociety/standard-footer#8]
I've made three releases so far, and every time I do it, I have to look up the SemVer guidelines from scratch, and then also look up the arcane git commands for creating and pushing a tag.

This stuff would fit nicely into a CONTRIBUTING.md file.

**Comment by zarino** [comment]
PS. The ["How we classify style changes" bit of this article](https://medium.com/design-optimizely/how-to-version-your-ui-library-1c7a1b7ee23a#86da) is a nice, clear outline of how SemVer can apply to style-based modules like our footer.

**Comment by zarino** [comment]
And the git workflow I always forget is:

```
# document the release in the changelog, 
# and increment the version number in the .scss file
# then…
git commit -m "Changelog for version 1.x.x"
git tag -a v1.x.x -m "Version 1.x.x of the Standard mySociety Footer."
git push origin HEAD --tags
```

**Comment by zarino** [comment]
Fixed by 1715d4f3fa11b9ed88173defd78089b7b4aeebc3
